server/graphql/v1/queries.js

```diff
@@ -317,8 +321,20 @@ const queries = {
       dateTo: { type: GraphQLString },
       /** @deprecated since 2018-11-29: Virtual cards now included by default when necessary */
       includeVirtualCards: { type: GraphQLBoolean },
+      fetchDataFromLedger: { type: GraphQLBoolean },
     },
     async resolve(_, args) {
+      const fetchDataFromLedger = args.fetchDataFromLedger || process.env.GET_TRANSACTIONS_FROM_LEDGER || false;
```

Comment:
If `args.fetchDataFromLedger` is explicitly false, it will use the value of `process.env.GET_TRANSACTIONS_FROM_LEDGER`. This is not what we want.